_posts/2018-02-08-rotating-signing-keys.adoc

```diff
@@ -18,15 +18,31 @@ toc::[]
 
 == Install the `garage-sign` tool
 
-On a debian machine, download the `garage-deploy` package and install it:
+On a Ubuntu 16.04 machine, download the `garage-deploy` package and install it:
```

Comment:
"an", oder?

Reply:
I mean, it depends on if you pronounce it OOO-buntu or YOU-buntu. I've always said the latter; google-fighting it seems to suggest about a 50-50 split. I'm gonna check if there's official guidance from canonical on this.

Reply:
Damn, they also have a roughly 50-50 split using `a` or `an` Ubuntu on *.ubuntu.com. 

Although I'm tempted to pull a Solomon and use `a` in one place and `an` in the other, I'm just going to rule this in favour of the answer I liked better in the first place (`a` ubuntu), since the evidence for changing it isn't that strong.

Reply:
I really don't care, I was just being pedantic.

That said, English seems to be the only language that by default assumes a "y" sound before a "u". I have no idea if that is true, but that is my experience. And since "Ubuntu" is definitely not an English word, I've always assumed it was not pronounced with standard English norms.

Reply:
I mean, yeah...but then again, we don't say "FOLKS-vah-gən" in English for VW. For better or worse, brand names that are taken from foreign words usually get pronounced with English phonetic norms, such as they are, when spoken in English.